lib/suma/api/commerce.rb

```diff
@@ -251,7 +254,7 @@ class OfferingProductEntity < BaseEntity
 
     expose :max_quantity
     expose :out_of_stock do |_|
```

Comment:
Move this to the model, and test it explicitly.

Reply:
@rgalanakis `out_of_stock` is reliant on the member's cart. How would this `out_of_stock` method fetch the necessary member? I haven't figured it out yet.

Reply:
Ah, I see. Ok, that's fine- this would be a pain to modify, either resulting in additional calculations, or a lot more complexity. Add a test though.

Reply:
FWIW, I would model `max_quantity_for` to `quantity_info_for` and return an object with fields like `max_quantity` and `out_of_stock`, rather than returning an integer. But I don't think that change is worth it for now.

Reply:
I agree it would add complexity. I'll go ahead and add tests for this.